bool inAutonomous
bool autoStart
bool pickupMaterial
bool transportToProduction
bool dropoffMaterial
bool transportToWarehouse

